RealmSwift/SwiftUI.swift

```diff
@@ -399,6 +408,12 @@ private class ObservableStorage<ObservedType>: ObservableObject where ObservedTy
         }
     }
     /// :nodoc:
+    @State public var `where`: ((Query<ResultType>) -> Query<ResultType>)? {
```

Comment:
Is this `where` overriding the results collection in case the user used `filter` before?. It may be helpful to add some documentation explaning that using `where` will override any use of filter, or that they are mutually excluding (only use one)